src/structures/MiniGames/PitInventoryItem.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Item class
+ */
+class PitInventoryItem {
+  /**
+   * @param {object} data Item data
+   */
+  constructor(data) {
+    /**
+     * Item ID
+     * @type {number}
+     */
+    this.itemId = data.id || 0;
+    /**
+     * Item count
+     * @type {number}
+     */
+    this.count = data.Count || 0;
+    /**
+     * Item name
+     * @type {string|null}
+     */
+    this.name = data?.tag?.display?.Name ? data.tag.display.Name.toString().replace(/§([1-9]|[a-f])|§/gm, '') : null;
+    /**
+     * Item lore
+     * @type {string|null}
+     */
+    this.lore = data?.tag?.display?.Lore ? data.tag.display.Lore.join('<br>') : null;
```

Comment:
my guy we are not in html
```suggestion
    this.lore = data?.tag?.display?.Lore ? data.tag.display.Lore.join('\n') : null;
```

Reply:
I feel that it should stay like this. `\n` will spit it into new lines and make it look bad and its always been `<br>` in use. If this changes we should change it in [SkyblockInventoryItem](https://github.com/Hypixel-API-Reborn/hypixel-api-reborn/blob/8eedeb4e4d9d8081bbaf396f414f2d0731fb6ac7/src/structures/SkyBlock/SkyblockInventoryItem.js#L30) as well

Reply:
<br> is html standard, \n is text 
showing a non-programmer "<br>" would get them confused, where as a correctly rendered \n (newline) ... would appear as newline.